crypto-core/src/main/java/com/palantir/crypto2/io/DecryptingSeekableInput.java

```diff
@@ -24,19 +24,19 @@
 import com.palantir.seekio.SeekableInput;
 import java.io.IOException;
 import javax.crypto.Cipher;
-import javax.crypto.CipherInputStream;
+import org.apache.commons.crypto.stream.CryptoInputStream;
 
 public final class DecryptingSeekableInput implements SeekableInput {
 
-    /** Size of the {@link CipherInputStream} internal buffer. */
-    private static final int CIPHER_INPUT_STREAM_BUFFER_SIZE = 512;
+    /** Size of the {@link CryptoInputStream} internal buffer. */
+    private static final int CRYPTO_INPUT_STREAM_BUFFER_SIZE = 8192;
```

Comment:
Unlike `CipherInputStream`, `CryptoInputStream` actually allows us to configure this. The default happens to be 8192, but it's likely worth playing with. I have a suspicion that we want it to align with `io.file.buffer.size`, but would need to read more code to be sure. @schlosna, interested to know if you have thoughts on how to determine the best value for this.